src/test/java/com/google/api/codegen/testdata/nodejs_test_library.baseline

```diff
@@ -0,0 +1,698 @@
+ * limitations under the License.
+ */
+
+var assert = require('assert');
+var service = require('../src/').v1();
+
+describe('LibraryServiceClient', function() {
+  describe('createShelf', function() {
+    it('Invokes createShelf without error', function(done) {
+      var client = service.libraryServiceClient();
+      // Mock request
+      var shelf = {};
+      var request = {
+          shelf : shelf
+      };
+
+      // Mock response
+      var name = 'name3373707';
+      var theme = 'theme110327241';
+      var internalTheme = 'internalTheme792518087';
+      var expectedResponse = {
+          name : name,
+          theme : theme,
+          internalTheme : internalTheme
+      };
+
+      // Mock Grpc layer
+      client._createShelf = function(actualRequest, options, callback) {
+        assert.deepStrictEqual(actualRequest, request);
+        callback(null, expectedResponse);
+      };
+
+      client.createShelf(request, function(err, response) {
+        assert.ifError(err);
+        assert.deepStrictEqual(response, expectedResponse);
+        done();
+      });
+    });
+  });
+
+  describe('getShelf', function() {
+    it('Invokes getShelf without error', function(done) {
+      var client = service.libraryServiceClient();
+      // Mock request
+      var formattedName = client.shelfPath("[SHELF_ID]");
+      var options = 'options-1249474914';
+      var request = {
+          name : formattedName,
+          options : options
+      };
+
+      // Mock response
+      var formattedName2 = client.shelfPath("[SHELF_ID]");
+      var theme = 'theme110327241';
+      var internalTheme = 'internalTheme792518087';
+      var expectedResponse = {
+          name : formattedName2,
+          theme : theme,
+          internalTheme : internalTheme
+      };
+
+      // Mock Grpc layer
+      client._getShelf = function(actualRequest, options, callback) {
+        assert.deepStrictEqual(actualRequest, request);
+        callback(null, expectedResponse);
+      };
+
+      client.getShelf(request, function(err, response) {
+        assert.ifError(err);
+        assert.deepStrictEqual(response, expectedResponse);
+        done();
+      });
+    });
+  });
+
+  describe('listShelves', function() {
+    it('Invokes listShelves without error', function(done) {
+      var client = service.libraryServiceClient();
+      // Mock request
+      var request = {};
+
+      // Mock response
+      var nextPageToken = '';
+      var shelvesElement = {};
+      var shelves = [shelvesElement];
+      var expectedResponse = {
+          nextPageToken : nextPageToken,
+          shelves : shelves
+      };
+
+      // Mock Grpc layer
+      client._listShelves = function(actualRequest, options, callback) {
+        assert.deepStrictEqual(actualRequest, request);
+        callback([expectedResponse.shelves, null, expectedResponse]);
+      };
+
+      client.listShelves(request, function(response) {
```

Comment:
Oh I see. I have changed the test code to be consistent with `callback(null, shelves);`
Done.